packages/core/src/components/contact-information/contact-information.scss

```diff
@@ -0,0 +1,35 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/typography";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/components/anchor";
+@use "~dso-toolkit/src/global/mixins/set-colors.mixin" as set-colors;
+
+:host {
+  @include anchor.hideIcons();
```

Comment:
Woeps, overheen gelezen: custom elements krijgen standaard `display: inline`, maar dit is absoluut een blok level component, en moet dus ook als dusdanig worden gestijld.